Makefile

```diff
@@ -133,6 +133,37 @@ cleanup-db-backup: ## Remove snapshot service and app
 paas-clean: ## Cleans up all files created for the PaaS deployment
 	cf logout
 
+.PHONY: create-db-cleanup-service
+create-db-cleanup-service: ## Create a db service for cleaning up latest dump to use in other environments.
+	./scripts/create-db-cleanup-service.sh
+
+.PHONY: check-db-cleanup-service
+check-db-cleanup-service: ## Get the status for the db cleanup service
+	@cf service digitalmarketplace_db_cleanup | grep -i 'status: ' | sed 's/^.*: //' | tr '[:lower:]' '[:upper:]'
+
+.PHONY: deploy-db-cleanup-app
+deploy-db-cleanup-app: ## Deploys the db cleanup app
+	@[ $$(cf target | grep -i 'space' | cut -d':' -f2) = "db-cleanup" ] || (echo "Error: This can only be run in the db-cleanup space" && exit 1)
+	cf push db-cleanup -o digitalmarketplace/db-cleanup --no-route --health-check-type none -i 1 -m 128M -c 'sleep 2h'
+	cf bind-service db-cleanup digitalmarketplace_db_cleanup
+	cf restage db-cleanup
+
+.PHONY: import-and-clean-db-dump
+import-and-clean-db-dump: virtualenv ## Connects to the db-cleanup service, imports the latest dump and cleans it.
+	@[ $$(cf target | grep -i 'space' | cut -d':' -f2) = "db-cleanup" ] || (echo "Error: This can only be run in the db-cleanup space" && exit 1)
+	VIRTUALENV_ROOT=${VIRTUALENV_ROOT} ./scripts/import-and-clean-db-dump.sh
+
+.PHONY: migrate-cleaned-db-dump
+migrate-cleaned-db-dump: ## Migrate the cleaned db dump to a target stage and sync with google drive.
+	@[ $$(cf target | grep -i 'space' | cut -d':' -f2) = "db-cleanup" ] || (echo "Error: This can only be run in the db-cleanup space" && exit 1)
+	./scripts/migrate-cleaned-db-to-target-stage.sh
+
+.PHONE: cleanup-db-cleanup
+cleanup-db-cleanup: ## Delete app and service created in the db cleanup procedure
+	@[ $$(cf target | grep -i 'space' | cut -d':' -f2) = "db-cleanup" ] || (echo "Error: This can only be run in the db-cleanup space" && exit 1)
+	cf delete -f db-cleanup
+	cf delete-service -f digitalmarketplace_db_cleanup
```

Comment:
That's a good question.

I guess that when the clean prod data is migrated to the stage, it could break? In that scenario we would could re-release the latest api release up to preview/staging which will run the latests migrations against the new data?